setup.py

```diff
@@ -28,7 +28,7 @@
     classifiers=['Development Status :: 4 - Beta',
                  'Environment :: Console',
                  'Intended Audience :: Developers',
-                 'License :: OSI Approved :: GNU General Public License (GPL)',
+                 'License :: The MIT License (MIT)',
```

Comment:
Nit: why to change licence here?

Reply:
I slightly modified it again as 'License :: OSI Approved :: MIT License' based on https://pypi.org/classifiers/.
The license is changed because the file `LICENSE` is `MIT`, see https://github.com/mailjet/mailjet-apiv3-python/blob/master/LICENSE, and https://spdx.org/licenses/MIT.html

Reply:
The initial commit says that the license is `GPL` (mistakenly?) but there is no license file https://github.com/mailjet/mailjet-apiv3-python/commit/19cf9a00a948e84de4842b51b0336e978f7a849f. Later, in **v1.0.3** the license file `LICENSE` was added https://github.com/mailjet/mailjet-apiv3-python/commit/08fd042acd03257ccc487568ec15d1fc534dfc57 and it's `MIT` since there

Reply:
@DanyilNefodov Do we need another approval for merging the PR or does changing the license name require a team agreement?

Reply:
No, I think we should just fine.

Feel free to merge as soon as you feel save

Reply:
Unfortunately, I don't have permission to merge PRs here